src/System.Xml.XmlSerializer/src/project.json

```diff
@@ -3,6 +3,7 @@
     "netstandard1.3": {
       "dependencies": {
         "System.Collections": "4.0.10",
+        "System.Collections.NonGeneric": "4.0.1-rc3-24015-00",
```

Comment:
Yes. Actually there are still places where we use Hashtable in emitted code. I have replaced them with ```Dictionary<object, object>``` for now. Due to the current object model, converting them to typed generic Dictionary may need further changes.